components/places/src/api/matcher.rs

```diff
@@ -112,30 +113,34 @@ fn looks_like_origin(string: &str) -> bool {
 
 /// The match reason specifies why an autocomplete search result matched a
 /// query. This can be used to filter and sort matches.
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Serialize)]
 pub enum MatchReason {
     Keyword,
     Origin,
     Url,
     PreviousUse,
     Bookmark,
+    // Hrm... This will probably make this all serialize weird...
     Tags(String),
```

Comment:
There are probably better ways to expose this...we could show this in the UI as a match for any tag, rather than a specific tag. (Or a tag ID, once we add a `moz_tags` table). But, since it's unused now, it's not worth untangling in this patch.